### PR TITLE
koordlet: remove dependencies of sqlite3

### LINKED ISSUE #1483: [proposal]clear code dependencies of sqlite3.
**What is your proposal**:
see https://github.com/koordinator-sh/koordinator/issues/1241 for full description.

**Why is this needed**:
remove ependencies of sqlite3 and clear code.

**Is there a suggested solution, if so, please add it**:

### PR DESCRIPTION
**Pull request by LambdaHJ**
### Ⅰ. Describe what this PR does

replace sqlite with tsdb, remove dependencies of sqlite3, clean deprecated interfaces and structs.
### Ⅱ. Does this pull request fix one issue?

fixes #1483 
### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: hack/mock-gen.sh [hunk 1]
```diff
@@ -23,10 +23,6 @@ LICENSE_HEADER_PATH="./hack/boilerplate/boilerplate.go.txt"
 cd $GOPATH/src/github.com/koordinator-sh/koordinator
 
 # generates gomock files
-mockgen -source pkg/koordlet/statesinformer/states_informer.go \
```
Comment (zwzhang0107) [inline hack/mock-gen.sh:26]: we should keep mockstatesinformer

Comment (LambdaHJ) [reply hack/mock-gen.sh:26]: pkg/koordlet/statesinformer/states_informer.go **no such file**

Comment (zwzhang0107) [reply hack/mock-gen.sh:26]: ops, it has been moved to pkg/koordlet/statesinformer/impl/states_informer.go

**Comment by zwzhang0107** [comment]
hi, please change the file path in mock_gen.sh to `pkg/koordlet/statesinformer/impl/states_informer.go`

**Review by hormes — APPROVED** [review]

**Pull request merged**